src/client/components/Sidebar/PostRecommendation.js

```diff
@@ -73,6 +74,7 @@ class PostRecommendation extends Component {
   getFilteredPosts = () => {
     const { match } = this.props;
     return this.state.recommendedPosts
+      .filter(post => formatter.reputation(post.author_reputation) > -1)
       .filter(post => post.permlink !== match.params.permlink)
```

Comment:
you don't need to iterate through the list again, you can create 1 filter function above that handles both filtering cases here